viewing_party/main.py

```diff
@@ -0,0 +1,247 @@
+
+def create_movie(title, genre, rating):
+    """
+    inputs: title (str), genre (str), rating (num)
+    outputs: a dictionary with 3 key-value pairs, which each correspond to one of the parameters
+    """
+
+    # initialize a dictionary
+    movie_dict = {}
+
+    # if ALL parameter values are truthy, add keys and values to dictionary and return dictionary
+    if title and genre and rating:
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+
+    # if ANY of the parameters return falsy, the function should return None
+    return None
```

Comment:
Consider dealing with the situation where you return None at the top since this is the edge case, such as:
```suggestion
    # if ANY of the parameters return falsy, the function should return None
    if not title or not genre or not rating:
         return None
```